llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -370,33 +338,39 @@ void addTableRow(util::SimpleTable &Table,
 
 // @param OutTables List of tables (one for each target) to output results
 // @param MD Module descriptor to save
+// @param OutputPrefix Prefix for the all generated outputs.
 // @param IRFilename filename of already available IR component. If not empty,
 //   IR component saving is skipped, and this file name is recorded as such in
 //   the result.
 void saveModule(std::vector<std::unique_ptr<util::SimpleTable>> &OutTables,
```

Comment:
```suggestion
void saveModule(const std::vector<std::unique_ptr<util::SimpleTable>> &OutTables,
```
I think it's not mutated.